### ISSUE TITLE
Use standards-track `allowedToPlay` if available

### ISSUE THREAD

**Issue by gkatsev** [video-dev/can-autoplay#25]
There's work to have a native method to know whether we're allowed to play, so, we should use that method if it's available. Mozilla said they're going to ship it in Firefox 63

https://github.com/whatwg/html/issues/3617

**Comment by caiogondim** [comment]
That's amazing news to see the lib is influencing an official spec ❤️ 
I would say let's wait until they settle on the API and first implementation, so we can make a test with a real browser.

**Comment by gkatsev** [comment]
Yeah, definitely best to wait till firefox ships it, though, we don't have to wait for long since it's [scheduled for late october](https://wiki.mozilla.org/Release_Management/Calendar).

Wanted to make sure everyone here was aware and that we were ready for the release :)

**Comment by NicolasSiver** [comment]
I was surprised from the beginning to find out that we have to do such a creative approach without any API. Browser vendors were so focused on all those user's concerns that they forgot about obvious stuff and video engineers.
